lib/modules/platform/azure/__snapshots__/index.spec.ts.snap

```diff
@@ -280,6 +280,7 @@ exports[`modules/platform/azure/index initRepo should initialise the config for
 {
   "defaultBranch": "defBr",
   "isFork": false,
+  "repoFingerprint": "02574de485149547c1a071aa7921da3d0afadcd6162f3bd49ba3ced29be589f8b9bac689fd0badb212bd21c3f48bd8566beaf31cdca2b083bd855808a9c129e2",
```

Comment:
we should get rid of those snapshots in separate PR

Reply:
Okay

Reply:
But let's do this after this one is merged, since it'll take a while